fetcher.go

```diff
@@ -261,17 +269,23 @@ func (p *pool) fetchResource(ctx context.Context, from string, resource string,
 	var resp *http.Response
 	saturnCallsTotalMetric.WithLabelValues(resourceType).Add(1)
 	startReq := time.Now()
-	resp, err = p.config.SaturnClient.Do(req)
+
+	for i := 0; i < 3; i++ {
+		resp, err = p.config.SaturnClient.Do(req)
+		if err == nil {
+			break
+		}
+	}
```

Comment:
this isn't something we want to do in prod, right?

Reply:
@willscott Why not ? What's wrong in retrying connections ? We're not increasing the timeout here and nodes that keep failing this will eventually get kicked out.

Reply:
because we're already in a loop doing retries one level up, where when we retry we retry to an alternate node.
adding a second loop here means we're now amplifying requests 9x instead of 3x without it being part of the parameter for how much we retry that's able to be controlled